enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionDefaultView/InteractiveQuestionDefaultView.tsx

```diff
@@ -188,27 +182,13 @@ export const InteractiveQuestionDefaultView = ({
               />
             </Group>
           </Group>
-        </Stack>
-      )}
+        )}
+      </Stack>
 
       <Box className={InteractiveQuestionS.Main} p="sm" w="100%" h="100%">
         <Box className={InteractiveQuestionS.Content}>
           {isEditorOpen ? (
-            <>
-              {/* Use the same horizontal padding as https://github.com/metabase/metabase/blob/98e2dcc7c8c7c9147f4a787cc7ed36eddde9c080/frontend/src/metabase/querying/notebook/components/Notebook/Notebook.tsx#L48 */}
```

Comment:
This extra back button could be removed, because we now always show the title which also includes the back button. This button was added in https://github.com/metabase/metabase/pull/60485, when we don't have the title before visualizing the query.

<img width="1351" height="650" alt="Image" src="https://github.com/user-attachments/assets/7650f29a-5ed1-4b6a-a8bf-e81c9f2b17b0" />